lib/rucio/transfertool/fts3_plugins.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+# Copyright European Organization for Nuclear Research (CERN) since 2012
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+from typing import Callable, Optional
+import json
+import sys
+
+from rucio.core.plugins import PolicyPackageAlgorithms
+from rucio.common.config import config_get_int, config_get_items
+from configparser import NoSectionError
+
+from rucio.common.exception import InvalidRequest
+
+
+class FTS3MetadataPlugin(PolicyPackageAlgorithms):
```

Comment:
Let's rename this to `FTS3TapeMetadataPlugin`, I think this is more accurate, since it is not meant for all kinds of metadata.